docs/how-to/deploy-to-heroku.md

```diff
@@ -0,0 +1,80 @@
+# Introduction
+
+In this guide, we will be looking at how to deploy bocadillo powered app to Heroku.
+You can see example code in [heroku-example](https://github.com/bocadilloproject/heroku-example) repo.
+
+### Bocadillo Application
```

Comment:
Careful about the casing — see suggestion below.

Also I think it's fine to use an `<h2>` here.

```suggestion
## Bocadillo application
```